networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-azure.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-azure.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-hosted-zone-azure_{context}"]
+= Creating DNS records on an public DNS zone for Azure by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Azure by using Red Hat External DNS Operator.
+
+.Procedure
+
+. Check the user. The user must have access to the `kube-system` namespace. If you don’t have the credentials, as you can fetch the credentials from the `kube-system` namespace to use the cloud provider client:
++
+[source,terminal]
+----
+$ oc whoami
+----
++
+.Example output
+[source,terminal]
+----
+system:admin
+----
+
+. Fetch the values from azure-credentials secret present in `kube-system` namespace.
++
+[source,terminal]
+----
+$ CLIENT_ID=$(oc get secrets azure-credentials  -n kube-system  --template={{.data.azure_client_id}} | base64 -d)
+$ CLIENT_SECRET=$(oc get secrets azure-credentials  -n kube-system  --template={{.data.azure_client_secret}} | base64 -d)
+$ RESOURCE_GROUP=$(oc get secrets azure-credentials  -n kube-system  --template={{.data.azure_resourcegroup}} | base64 -d)
+$ SUBSCRIPTION_ID=$(oc get secrets azure-credentials  -n kube-system  --template={{.data.azure_subscription_id}} | base64 -d)
+$ TENANT_ID=$(oc get secrets azure-credentials  -n kube-system  --template={{.data.azure_tenant_id}} | base64 -d)
+----
+
+. Login to azure with base64 decoded values:
++
+[source,terminal]
+----
+$ az login --service-principal -u "${CLIENT_ID}" -p "${CLIENT_SECRET}" --tenant "${TENANT_ID}"
+----
+
+. Get the routes to check the domain:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example output
+[source,terminal]
+----
+openshift-console          console             console-openshift-console.apps.test-azure.qe.azure.devcluster.openshift.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.test-azure.qe.azure.devcluster.openshift.com                     downloads           http    edge/Redirect          None
+----
```

Comment:
Let's follow the GCP part and update the domain for Azure as well. Please replace `test-azure.qe.azure.devcluster.openshift.com` to `test.azure.example.com` for both routes.

Reply:
Addressed